### PR TITLE
Prisma config update to work with new supabase supavisor

### PR DESCRIPTION
**Pull request by pyrotank41**
## Summary by CodeRabbit

- **New Features**
    - Introduced support for using Supabase as a database option, including new configuration parameters.
- **Refactor**
    - Added validation for database URLs to ensure compatibility and security.
- **Documentation**
    - Updated documentation to reflect the changes in database configuration.

### CODE REVIEW

File: packages/prisma/postgresql/schema.prisma [hunk 1]
```diff
@@ -4,7 +4,8 @@ generator client {
 
 datasource db {
   provider = "postgresql"
-  url      = env("DATABASE_URL")
+  url       = env("DATABASE_URL")
+  directUrl = env("DIRECT_URL")
```
Comment (baptisteArno) [inline packages/prisma/postgresql/schema.prisma:8]: Can we change the name of the variable to `DATABASE_DIRECT_URL`?

**Review by baptisteArno — CHANGES_REQUESTED** [review]

**Review by pyrotank41 — COMMENTED** [review]
done! does this work?

**Review by baptisteArno — APPROVED** [review]

**Comment by baptisteArno** [comment]
I prefer not to merge this as long as this issue is not tackled: https://github.com/prisma/prisma/issues/23221

**Comment by wesleybpereira** [comment]
Is it now possible to install with Supabase?

**Comment by jwalsh-vori** [comment]
I had to make these changes to my fork to enable my self-hosted version to work with Supabase. Can we get this merged in, and maybe add an ENV VAR for SUPABASE_ENABLED to allow us to use Supabase as a DB and also S3 compatible storage?

**Review by Messhias — APPROVED** [review]

**Comment by baptisteArno** [comment]
Closing it in favor of: https://github.com/baptisteArno/typebot.io/issues/1712

**Pull request closed without merging**